### ISSUE TITLE
blockOnMeta: fresh meta

### ISSUE THREAD

**Issue by m0davis** [agda/agda#1794]
Using Ulf's new-reflection branch and agda-prelude, I can generate an internal error. Replace the induction-example of Tactic.Nat with the following, then C-c C-r in the hole. The location of the error is src/full/Agda/TypeChecking/Rules/Term.hs:1080.

```
  induction-example : ∀ n → sum (downFrom n) * 2 ≡ n * (n + 1)
  induction-example = {!simplify-goal!}
```

commits used:
https://github.com/UlfNorell/agda/tree/daac59336d8d4d5c00083e8976fc1e177f73cdec 
https://github.com/UlfNorell/agda-prelude/tree/657a0c88ab3d94a62dbb74b9373c71e654d7d329

**Comment by UlfNorell** [comment]
This is a known problem. What happens is that the tactic blocks on a meta (`blockOnMeta`) that was created during the execution of the script. I'm not sure yet of how to handle this case.

**Comment by UlfNorell** [comment]
Fixed by a630a7e. Solves it by blocking on one of the metavariables in the goal type instead of on the fresh meta. If there are no metas in the goal the macro call will be left yellow.

**Comment by asr** [comment]
I added the 2.4.4 milestone.
